docs/integrations/gitlab/org.md

```diff
@@ -65,11 +65,52 @@ catalog:
       yourProviderId:
         host: gitlab.com
         orgEnabled: true
-        group: org/teams # Optional. Must not end with slash. Accepts only groups under the provided path (which will be stripped)
+        group: org/teams # Required for gitlab.com when `orgEnabled: true`. Optional for self managed. Must not end with slash. Accepts only groups under the provided path (which will be stripped)
         groupPattern: '[\s\S]*' # Optional. Filters found groups based on provided pattern. Defaults to `[\s\S]*`, which means to not filter anything
 ```
 
+### Groups
+
 When the `group` parameter is provided, the corresponding path prefix will be
 stripped out from each matching group when computing the unique entity name.
 e.g. If `group` is `org/teams`, the name for `org/teams/avengers/gotg` will be
 `avengers-gotg`.
+
+For gitlab.com, when `orgEnabled: true`, the `group` parameter is required in
+order to limit the ingestion to a group within your organisation. `Group`
+entities will only be ingested for the configured group, or it's descendant groups,
+but not any ancestor groups higher than the configured group path. Only groups
+which contain members will be ingested.
```

Comment:
As for the level within the hierarchy, it made sense that if you configure a group path, then the only groups ingested would be at that group level, or lower. Rather than in the other direction, higher up the hierarchy. Regarding ingestion of only groups with members, this was my understanding of how this always worked, but let me double check on that assumption.